oci/runtime_oci.go

```diff
@@ -35,6 +35,9 @@ const (
 
 	// Command line flag used to specify the run root directory
 	rootFlag = "--root"
+
+	// default run root to be used for OCI runtime if it is not set by handler
+	defaultRunRoot = "/run/runc"
```

Comment:
If we give this PR precedence to #2490, then I'd have a fix in mind to avoid the duplication of that variable. :)

Reply:
I like the sounds of that!